sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -46,6 +46,11 @@ export const DefaultConnectOptionsConstants = {
 };
 
 export const API_VERSION = "2023-10-01-preview";
+type ErrorMessages = {
```

Comment:
use the common/types file for all the types.

Reply:
sure.